src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/relationships/relationships.js

```diff
@@ -112,193 +109,110 @@ class RelationshipsUI extends Component {
-                you delete this index pattern, these saved searches will not
-                longer work properly."
-              />);
-            }
-            break;
-        }
-
-        items.push(
-          <Fragment key={type}>
-            <EuiDescriptionListTitle style={{ marginBottom: '1rem' }}>
-              <EuiCallOut color={calloutColor} title={calloutTitle}>
-                <p>{calloutText}</p>
-              </EuiCallOut>
-            </EuiDescriptionListTitle>
-            <EuiInMemoryTable
-              items={list}
-              columns={[
-                {
-                  width: '24px',
-                  render: () => (
-                    <EuiToolTip
-                      position="top"
-                      content={getSavedObjectLabel(type)}
-                    >
-                      <EuiIcon
-                        aria-label={getSavedObjectLabel(type)}
-                        size="s"
-                        type={getSavedObjectIcon(type)}
-                      />
-                    </EuiToolTip>
-                  ),
-                },
-                {
-                  name: intl.formatMessage({
-                    id: 'kbn.management.objects.objectsTable.relationships.columnTitleName', defaultMessage: 'Title'
-                  }),
-                  field: 'title',
-                  render: (title, item) => (
-                    <EuiLink href={`${getEditUrl(item.id, type)}`}>
-                      {title}
-                    </EuiLink>
-                  ),
-                },
-                {
-                  name: intl.formatMessage({
-                    id: 'kbn.management.objects.objectsTable.relationships.columnActionsName', defaultMessage: 'Actions'
-                  }),
-                  actions: [
-                    {
-                      name: intl.formatMessage({
-                        id: 'kbn.management.objects.objectsTable.relationships.columnActions.inAppName',
-                        defaultMessage: 'In app'
-                      }),
-                      description: intl.formatMessage({
-                        id: 'kbn.management.objects.objectsTable.relationships.columnActions.inAppDescription',
-                        defaultMessage: 'View this saved object within Kibana'
-                      }),
-                      icon: 'eye',
-                      onClick: object => goInApp(object.id, type),
-                    },
-                  ],
-                },
-              ]}
-              pagination={true}
+              id="kbn.management.objects.objectsTable.relationships.relationshipsTitle"
+              defaultMessage="Related objects"
             />
-          </Fragment>
-        );
-      }
-    }
-
-    return <EuiDescriptionList>{items}</EuiDescriptionList>;
+          </h3>
+        </EuiTitle>
+        <EuiInMemoryTable
+          items={relationships}
+          columns={columns}
```

Comment:
Maybe add a column to indicate if the relationship is to or from this saved object.